src/docs/product/integrations/integration-platform/ui-components/issue-link.mdx

```diff
@@ -0,0 +1,145 @@
+
+## Issue Link Schema
+
+```json {filename:schema.json}
+{
+  "elements": [
+    {
+      "type": "issue-link",
+      "link": {
+        "uri": <String>,
+        "required_fields": <Array<FormField>>,
+        "optional_fields": <Array<FormField>>
+      },
+      "create": {
+        // Same as "link" schema
+      }
+    }
+  ]
+}
+```
+
+### Attributes of the Link and Create Fields
+
+- `uri` - (Required) The URI to request when the User submits the Link/Create Issue form.
+- `required_fields` - (Required) List of [FormField](/product/integrations/integration-platform/ui-components/formfield) components the User is required to complete.
+- `optional_fields` - (Optional) List of [FormField](/product/integrations/integration-platform/ui-components/formfield) components the User may complete.
+
+
+## Issue Link Request and Response
+
+When a user attempts to create or link an issue, we will send a request to your service based on the provided `uri`.
+
+### Request Format
+
+We send a `POST` request, so all the information is stored in the body.
+
+### Schema
+
+```json {filename:schema.json}
+{
+  "elements": [
+    {
+      "fields": <Object>,
+      "installationId": <String>,
+      "issueId": <String>,
+      "webUrl": <String>,
+      "project": {"slug": <String>, "id": <String>},
+      "actor": {"type": "user", "name": <String>, "id": <String>},
+    }
+  ]
+}
+```
+
+### Attributes
+
+- `fields` - Key/value pairs of the data in the form fields
+- `installationId` - The ID of the installation associated with the request
+- `issueId` - The ID of the issue in Sentry
+- `webUrl` - The URL of the issue in Sentry
+- `project`
+  - `slug` - The slug of the project in Sentry
+  - `id` - The ID of the project in Sentry
+- `actor`
+  - `name` - The name of the user in Sentry
+  - `id` - The ID of user in Sentry
+
+### Response Format
+
+### Schema
+When creating or linking an issue, the response format _must_ have the following schema:
+
+```json
+{
+  "webUrl": <String>,
+  "project": <String>,
+  "identifier": <String>,
+}
+```
+
+### Attributes
+
+- `webUrl`- The URL of the linked issue in your project management system
+- `project`- The first part of the displayed issue that should probably be associated with a project in your system
+- `identifier`- The second part of the displayed issue that should be unique to the project
+
+When an external issue is created or linked in [sentry.io](https://sentry.io), a display name is shown that links back to the service where it was either created or linked. The display name is composed of two pieces: the `project` and the `identifier`. A hash (#) connects each piece. Here's an example of what it looks like in [sentry.io](https://sentry.io):
+
+![Sentry's sidebar button illustrating that an issue is linked.](../../issue-tracking/shortcut/shortcut_story_in_linked_issues_82.png)
```

Comment:
This section of info should be located just after the modal form image, not under this heading.